hoomd/hpmc/GSDHPMCSchema.h

```diff
@@ -115,21 +119,27 @@ struct gsd_shape_schema<hpmc::sph_params>: public gsd_schema_hpmc_base
             )
         {
 
-        std::string path = name + "radius";
+        std::string path_o = name + "orientable";
         std::vector<float> data;
+        std::string path = name + "radius";
+        std::vector<uint32_t> orientableflag;
         bool state_read = true;
         if(m_exec_conf->isRoot())
             {
             data.resize(Ntypes, 0.0);
+            orientableflag.resize(Ntypes);
             if(!reader->readChunk((void *) &data[0], frame, path.c_str(), Ntypes*gsd_sizeof_type(GSD_TYPE_FLOAT), Ntypes))
                 state_read = false;
+            if(!reader->readChunk((void *) &orientableflag[0], frame, path_o.c_str(), Ntypes*gsd_sizeof_type(GSD_TYPE_UINT32), Ntypes))
+                state_read = false;
```

Comment:
This makes it an error to read state values from existing files.

We need to bump the HOOMD schema version in glotzerlab/gsd#22 to 1,3 and use the following logic here:
```
if (schema <= gsd_make_version(1,2))
    orientable = [0,0,0,0...]
else
    read orientable chunk from file and set state_read appropriately
```